src/Storages/MergeTree/MergeTreeSettings.h

```diff
@@ -203,6 +202,7 @@ struct Settings;
     M(MaxThreads, max_part_loading_threads, 0, "Obsolete setting, does nothing.", 0) \
     M(MaxThreads, max_part_removal_threads, 0, "Obsolete setting, does nothing.", 0) \
     M(Bool, use_metadata_cache, false, "Obsolete setting, does nothing.", 0) \
+    M(Bool, allow_suspicious_indices, false, "Obsolete setting, does nothing.", 0) \
```

Comment:
It's a backward incompatible change because some users have already enabled that merge tree setting (and they expect that it does something)